tests/h/views/accounts_test.py

```diff
@@ -660,6 +660,29 @@ def test_get_sets_subscriptions_data_in_form(self,
             'notifications': set(['reply']),
         })
 
+    def test_get_for_user_without_email_address(self,
```

Comment:
Can you make this test name match our test-function-naming template so I can tell what it's testing more easily?

our naming guidelines are something like:
`test_it_[does_this/has this result]_[when/if]_[condition]`